linkerd/protocol/http/src/e2e/scala/io/buoyant/linkerd/protocol/HttpTlsEndToEndTest.scala

```diff
@@ -4,11 +4,12 @@ import java.io.{File, InputStream}
 import java.net.InetSocketAddress
 import java.nio.file.StandardCopyOption.REPLACE_EXISTING
 import java.nio.file.{Files, Paths}
+
```

Comment:
please remove spaces and alphasort imports

Reply:
Fixing